docs/core/compatibility/sdk/8.0/debugsymbols.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Breaking change: Setting DebugSymbols to false disables PDB generation"
+description: Learn about the breaking change in the .NET SDK where setting DebugSymbols to false disables PDB generation.
+no-loc: [DebugSymbols]
+ms.date: 07/30/2024
+---
+# Setting DebugSymbols to false disables PDB generation
+
+The existing [MSBuild documentation](/visualstudio/msbuild/common-msbuild-project-properties) says that specifying `DebugSymbols=false` on the command line means that program database (*.pdb*) files aren't generated. However, that wasn't true before .NET 8. The behavior has been updated such that setting `DebugSymbols` to `false` now suppresses PDB generation by changing `DebugType` to `None`.
+
+If you currently have a script that includes `-p:DebugSymbols=false`, you might expect PDBs to be created. If you upgrade to .NET 8 or a later version, that behavior will change.
```

Comment:
If people have a script that includes `-p:DebugSymbols=false` and expect PDBs to be created, they probably don't realize they have `-p:DebugSymbols=false`. I don't know how to express that, though...this is correct, but I could imagine someone (who has that set) reading this and saying, "well I don't have that, so this doesn't apply" when it does.